src/views/search/list/conversation/search-conversation-list-item.tsx

```diff
@@ -0,0 +1,317 @@
+	deselectAll: () => void;
+};
+
+export const SearchConversationListItem: FC<SearchConversationListItemProps> = memo(
+	function ConversationListItem({
+		conversationId,
+		selecting,
+		active,
+		activeItemId,
+		toggle,
+		selected,
+		deselectAll
+	}) {
+		const conversation = useConversationById(conversationId);
+		const dispatch = useAppDispatch();
+		const [open, setOpen] = useState(false);
+		const accounts = useUserAccounts();
+		const messages = useConversationMessages(conversationId);
+		const { createWindow } = useGlobalExtraWindowManager();
+		const conversationStatus = useConversationStatus(conversationId);
+		const tagsFromStore = useTags();
+		const tags = useMemo(
+			() =>
+				uniqBy(
+					reduce(
+						tagsFromStore,
+						(acc: Array<Tag>, v) => {
+							if (includes(conversation.tags, v.id)) {
+								acc.push({
+									...v,
+									// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+									// @ts-ignore
+									color: ZIMBRA_STANDARD_COLORS[v.color || 0].hex
+								});
+							} else if (conversation.tags?.length > 0 && !includes(conversation.tags, v.id)) {
+								forEach(
+									filter(conversation.tags, (tn) => tn.includes('nil:')),
+									(tagNotInList) => {
+										acc.push({
+											id: tagNotInList,
+											name: tagNotInList.split(':')[1],
+											color: 1
+										});
+									}
+								);
+							}
+							return acc;
+						},
+						[]
+					),
+					'id'
+				),
+			[conversation.tags, tagsFromStore]
+		);
+
+		const zimbraPrefMarkMsgRead = useUserSettings()?.prefs?.zimbraPrefMarkMsgRead !== '-1';
+		const participantsString = useMemo(
+			() =>
+				reduce(
+					uniqBy(conversation.participants, (em: Participant) => em.address),
+					(acc, part) => trimStart(`${acc}, ${participantToString(part, accounts)}`, ', '),
+					''
+				),
+			[conversation.participants, accounts]
+		);
+
+		const expandConversation = useCallback(
+			(e) => {
+				e.preventDefault();
+				setOpen((currentlyOpen) => {
+					if (
+						!currentlyOpen &&
+						conversationStatus !== API_REQUEST_STATUS.fulfilled &&
+						conversationStatus !== API_REQUEST_STATUS.pending
+					) {
+						retrieveConversation(conversationId);
+					}
+					return !currentlyOpen;
+				});
+			},
+			[conversationId, conversationStatus]
+		);
+
+		const _onClick = useCallback(
+			(e) => {
+				if (!e.isDefaultPrevented()) {
+					if (conversation?.read === false && zimbraPrefMarkMsgRead) {
+						setConversationsReadWithCallback({
+							ids: [conversationId],
+							value: false,
+							dispatch,
+							deselectAll
+						}).onClick();
+					}
+					pushHistory(`conversation/${conversationId}`);
+				}
+			},
+			[conversation?.read, conversationId, zimbraPrefMarkMsgRead, dispatch, deselectAll]
+		);
+
+		const _onDoubleClick = useCallback(
+			(e) => {
+				if (e.isDefaultPrevented()) {
+					return;
+				}
+
+				const { id, isDraft, parent } = conversation.messages[0];
+				if (isDraft) {
```

Comment:
Do we want to edit a draft while in search module??? In mails it makes sense, but here I don't know